api/src/services/files.ts

```diff
@@ -151,7 +150,7 @@ export class FilesService extends ItemsService {
 			});
 		}
 
-		const parsedURL = url.parse(fileResponse.request.res.responseUrl);
+		const parsedURL = new URL(fileResponse.request.res.responseUrl);
```

Comment:
`new URL` is not a drop-in replacement for `url.parse`

Reply:
Okay, thanks! I'm going to omit this change in the new pull request.